code/dynamic_programming/src/Optimal_Binary_Search_Tree/Optimal_BST.cpp

```diff
@@ -0,0 +1,68 @@
+#include <vector>
+#include <iostream>
+
+using std::vector;
+using std::endl;
+using std::cin;
+using std::cout;
+/* 
```

Comment:
You should put this comment at the beginning of the file, and add this to it : Part of Cosmos by OpenGenus.